backend/asmlink.ml

```diff
@@ -230,20 +229,26 @@ let force_linking_of_startup ~ppf_dump =
   Asmgen.compile_phrase ~ppf_dump
     (Cmm.Cdata ([Cmm.Csymbol_address "caml_startup"]))
 
-let make_globals_map units_list ~crc_interfaces =
-  let crc_interfaces = String.Tbl.of_seq (List.to_seq crc_interfaces) in
+let make_globals_map units_list =
+  (* The order in which entries appear in the globals map does not matter
+     (see the natdynlink code).
+     Note that [!interfaces] can contain duplicates.
```

Comment:
This line doesn't seem to make sense since `interfaces` is a set now.

Reply:
Ah, that is a left-over from when it was a list.